airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -126,6 +126,7 @@ definitions:
       - stream_slicers
     properties:
       type:
+        type: string
```

Comment:
:information_source: We'll always need a `type: string` beside the single element `enum`, to make sure Orval can generate the actual type properly, and not render it as `unknown`. cc @flash1293 @lmossman